main_test.go

```
@@ -28,6 +28,7 @@ func TestProcessAutoTagDocuments(t *testing.T) {
 	require.NoError(t, err)
 	correspondentTemplate, err = template.New("correspondent").Funcs(sprig.FuncMap()).Parse("")
 	require.NoError(t, err)
+	createdDateTemplate, err = template.New("created_date").Funcs(sprig.FuncMap()).Parse("")
```

golangci-lint (1.64.8)

ineffectual assignment to err

(ineffassign)